Lib/multiprocessing/managers.py

```diff
@@ -653,9 +686,9 @@ def register(cls, typeid, callable=None, proxytype=None, exposed=None,
                            getattr(proxytype, '_method_to_typeid_', None)
 
         if method_to_typeid:
-            for key, value in list(method_to_typeid.items()):
-                assert type(key) is str, '%r is not a string' % key
-                assert type(value) is str, '%r is not a string' % value
+            for key, value in list(method_to_typeid.items()): # isinstance?
```

Comment:
I think these 3 lines should not be changed.

Reply:
OK.